src/rust/organization-management/rebuild-sqlx.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
You'll want to add a `BUILD` file to `src/rust/organization-management` with the following contents:

```
shell_sources()
```
This will ensure that the shell scripts in this directory are formatted and linted (and even tested, if you like!) by Pants.